redbot/core/_sharedlibdeprecation.py

```diff
@@ -21,9 +21,9 @@ def find_spec(self, fullname, path, target=None) -> None:
         if parts[0] != "cog_shared" or len(parts) != 2:
             return None
         msg = (
-            "One of cogs uses shared libraries which are"
+            "One of the cogs uses shared libraries which are"
             " deprecated and scheduled for removal in the future.\n"
-            "You should inform author of the cog about this message."
+            "You should inform the author of the cog about this message."
```

Comment:
It could be multiple authors. I think the right one would be `You should inform author(s) of the cog about this message.`

Reply:
gud point

Reply:
> It could be multiple authors. I think the right one would be `You should inform author(s) of the cog about this message.`

`You should inform the author(s) of the cog...` IMO